app/src/main/java/org/lineageos/selfie/MainActivity.kt

```diff
@@ -304,6 +342,8 @@ class MainActivity : AppCompatActivity() {
                     Toast.LENGTH_SHORT
                 ).show()
                 finish()
+            } else if (!allLocationPermissionsGranted()) {
+                ActivityCompat.requestPermissions(this, REQUIRED_PERMISSIONS_LOCATION, 0)
```

Comment:
No code for GPS?

Reply:
![image](https://user-images.githubusercontent.com/6830979/184314945-b47a7b4b-1e0f-4977-99f0-1971ef501550.png)

Reply:
Ayy [1]